packages/workflow/src/components/style.scss

```diff
@@ -0,0 +1,229 @@
+@use "@wordpress/base-styles/mixins" as *;
+@use "@wordpress/base-styles/variables" as *;
+@use "@wordpress/base-styles/colors" as *;
+@use "sass:color";
+
+// dirty hack to clean up modal
+.workflows-workflow-menu {
+	border-radius: $grid-unit-05;
+	width: calc(100% - #{$grid-unit-40});
+	margin: auto;
+	max-width: 400px;
+	position: relative;
+	top: calc(5% + #{$header-height});
+
+	@include break-small() {
+		top: calc(10% + #{$header-height});
+	}
+
+	.components-modal__content {
+		margin: 0;
+		padding: 0;
+	}
+}
+
+.workflows-workflow-menu__overlay {
+	display: block;
+	align-items: start;
+}
+
+.workflows-workflow-menu__header {
+	display: flex;
```

Comment:
Could we use components like hstack here and remove some of these styles? Ideally we would reduce a little bit the amount of custom styling we have an try to use the components if we can.